src/switch_exporter/server.py

```diff
@@ -64,6 +64,9 @@ def get_arguments() -> argparse.Namespace:
     parser.add_argument(
         '--password', default='monitor',
         help='Password on switches')
+    parser.add_argument(
+        '--keyfile',
+        help='SSH key for switches')
```

Comment:
I'm not seeing anywhere that this argument gets used. `Switch.keyfile` is just set to `[]` and never changed.

Reply:
Added https://github.com/ska-sa/switch_exporter/commit/f57b1eb0b2f702c0cb30860c0d2a5ce639970874